src/Polyfill/Polyfill.targets

```diff
@@ -17,6 +17,9 @@
   <PropertyGroup Condition="$(PolyPublic) == 'true' ">
     <DefineConstants>$(DefineConstants);PolyPublic</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="$(PolyUseEmbeddedAttribute) == 'true' ">
+    <DefineConstants>$(DefineConstants);PolyUseEmbeddedAttribute</DefineConstants>
+  </PropertyGroup>
```

Comment:
We need to verify that `PolyPublic` is not set as that would be contradictionary.

Reply:
Done.